CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 ## **[Unreleased]**
 
 ### Added
+- [#2123](https://github.com/wasmerio/wasmer/pull/2123) `libwasmer` comes with all available compilers per targets instead of Cranelift only.
```

Comment:
```suggestion
- [#2123](https://github.com/wasmerio/wasmer/pull/2123) `libwasmer` comes with all available compilers per target instead of Cranelift only.
```

Reply:
Thanks!